static/js/compose.js

```diff
@@ -200,6 +200,15 @@ export function update_closed_compose_buttons_for_stream() {
     update_conversation_button(text_conversation, title_conversation);
 }
 
+export function update_closed_compose_buttons_for_recent_topics() {
+    update_closed_compose_buttons_for_private();
+    // Hide reply button since there is no message
+    // in recent topics to reply to.
+    // TODO: Allow user to reply to currently
+    // focused topic / stream in the topics table.
+    $("#left_bar_compose_reply_button_big").hide();
+}
```

Comment:
Ideally, we should put this in the new `compose_closed_ui.js`, along with the adjacent functions; do you want to do a prep commit to bulk-move this bundle of functions there?

Reply:
Seems like you did it and I removed this function actually.

Reply:
I just moved the click handlers from `click_handlers.js`, not this bundle of functions.

Reply:
(But if we no longer need this function, certainly we can consider that move a follow-up cleanup)

Reply:
Okay.